keps/sig-network/3453-minimize-iptables-restore/README.md

```diff
@@ -244,22 +233,14 @@ appropriate code for that.
 
 #### Beta
 
-- If we decide that we need to implement the "partial sync gives
-  different result than full sync" metric, then we will not go to Beta
-  until at least one release has passed with that metric available in
-  Alpha.
-
-- No new bugs
-- Feature actually improves performance (eg, in the `gce-5000Nodes`
-  periodic job).
-- Real-world users trying out the Alpha feature have reported both
-  (a) no new bugs, and (b) improved performance.
-- General e2e tests modified to check the "partial sync failures"
-  metric at some point or points during the test run, TBD.
+(We have decided to skip Beta.)
```

Comment:
The risk here is being able to disable it, if needed (your point from the call).  While we CAN make a disableable GA, it's unusual and has little ROI here.  So I think we should either: 

a) Do a regular locked-on GA (YOLO!)
b) Do a regular beta

There's not much cost to beta - it will be on by default and disableable and it follows the normal flow.

Thoughts?

Reply:
I strongly believe we should make a beta.

As Tim pointed out, I don't think there is much of additional overhead.
Given we believe we're code-complete, Beta will effectively only be changing a feature gate.
And it gives us ability to disable that in case of problems (as opposed to GA that would be locked-on).

Reply:
@thockin I'm fine going through beta but don't the arguments in favor of it contradict your new "we should get rid of beta" proposal? :slightly_smiling_face: